client/src/call-customer/components/board.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState, useRef } from 'react'
```

Comment:
As a best practice in React, you may always want to call components Uppercase i.e. Board.tsx